charts/flux2/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: flux2
 description: A Helm chart for flux2
 type: application
-version: 0.19.2
-appVersion: 0.30.2
+version: 0.19.3
```

Comment:
Given changes that add new features / capabilities, we should increment the minor version rather than the patch.